### PR TITLE
feat: added card component

### PR DESCRIPTION
**Pull request by abhisekgupta001**
Hi, added card component.

please do let me know if I am missing any particular needed type of card.

- profile card [solid and outlined]
- product card [with badge, is vertical, has shadows]
- dialog card with dismiss button , has shadows and is horizontal
- card with gradient text overlay with a stylized badge and has shadows.

### CODE REVIEW

File: public/components/card/card.css [hunk 1]
```diff
@@ -1,9 +1,213 @@
...
+    border-radius: 5px;
+
+}
+.btn-wishlist:hover{
+    color: var(--primary-color-light);
+}
+
+/* ------------------dismiss card -------------- */
+.card-with-dismiss{
+    width: 300px;
```
Comment (znjs) [inline public/components/card/card.css]: can use relative units

File: public/components/card/card.css [hunk 2]
```diff
@@ -1,9 +1,213 @@
...
+    gap: 1rem;
+}
+
+.btn-add-to-cart{
+    margin:0;
+    padding:0.2rem 0.4rem;
+    border-radius: 5px;
+}
+.btn-add-to-cart:hover{
+background-color: var(--primary-color-light);
```
Comment (znjs) [inline public/components/card/card.css:146]: can change this color on hover button text and background color is blending

**Review by znjs — APPROVED** [review]

**Review by JowelTisso — APPROVED** [review]
Suggestion: Give more spacing in the UI.

File: public/components/card/card.css (hunk 1 above)
Comment (abhisekgupta001) [reply public/components/card/card.css]: changed. Thanks for the suggestion

**Pull request merged**